bugs/gh-529-539.txt

```diff
@@ -0,0 +1,2734 @@
+            --platform-fault-domain-count 2  \
+            --platform-update-domain-count 2
+
+        az network nic create \
+            --name $AZURE_NIC_NAME-$UBUNTU_RELEASE \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --network-security-group $AZURE_NETWORK_SECURITY_GROUP \
+            --vnet-name $AZURE_VNET_NAME \
+             --accelerated-networking true \
+            --subnet sruSubnet \
+            --private-ip-address-version IPv4 \
+            --lb-address-pools dsLbBackEndPool_v4_$UBUNTU_RELEASE  \
+            --lb-name dsLB_$UBUNTU_RELEASE  \
+            --public-ip-address vmPublicIP-IPv4-$UBUNTU_RELEASE
+
+        az network nic create \
+            --name ${AZURE_NIC_NAME}2-$UBUNTU_RELEASE \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --network-security-group $AZURE_NETWORK_SECURITY_GROUP \
+            --vnet-name $AZURE_VNET_NAME \
+            --accelerated-networking true \
+            --subnet sruSubnet \
+            --private-ip-address-version IPv4
+
+        # Add IPv6 on primary nic
+        az network nic ip-config create \
+            --name $AZURE_NIC_NAME-$UBUNTU_RELEASE-v6 \
+            --nic-name $AZURE_NIC_NAME-$UBUNTU_RELEASE \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --vnet-name $AZURE_VNET_NAME \
+            --subnet sruSubnet \
+            --lb-address-pools dsLbBackEndPool_v6_$UBUNTU_RELEASE  \
+            --lb-name dsLB_$UBUNTU_RELEASE \
+            --private-ip-address-version IPv6
+
+    fi
+    az vm show --name $name -g $AZURE_RESOURCE_GROUP > /dev/null 2>&1 ||
+        az vm create --name=$name \
+            --image=$image_version:latest \
+            --admin-username=root --admin-username=ubuntu \
+            -g $AZURE_RESOURCE_GROUP --ssh-key-value $SSH_KEY \
+            $AZURE_BOOT_DIAG_ARG \
+            --custom-data $SAMPLE_CLOUDCONFIG $nics
+    IP=$(az vm list-ip-addresses --name $name | jq -r '.[] | .virtualMachine.network.publicIpAddresses[].ipAddress'| awk '{printf "ubuntu@%s", $1}')
+
+    echo "Created $name: $IP"
+
+    sleep 30
+    retry_ssh_command $IP "cloud-init status --wait --long" 5
+    ssh "${SSHOPTS[@]}" $IP -- cloud-init status --wait --long
+    echo "--- TEST: EXPECT NO KVP dump logs in cloud-init.log" #bbsw
+    ssh "${SSHOPTS[@]}" $IP -- ! grep -i dump /var/log/cloud-init.log  || echo "FAILURE: Found KVP dump logs in /var/log/cloud-init.log"
+    ssh "${SSHOPTS[@]}" $IP -- dpkg-query --show cloud-init
+    ssh "${SSHOPTS[@]}" $IP -- sudo cat /run/cloud-init/result.json
+    ssh "${SSHOPTS[@]}" $IP -- systemd-analyze
+    ssh "${SSHOPTS[@]}" $IP -- systemd-analyze blame | grep cloud-init
+    echo "REBOOTING TO Confirm boot extra timeout across reboot LP: #1830740"  #bbsw
+    ssh "${SSHOPTS[@]}" $IP -- sudo reboot || true
+    ssh "${SSHOPTS[@]}" $IP -- cloud-init status --wait --long
+    echo "--- TEST: BEGIN EXPECT Long boot time LP: #1830740"
```

Comment:
Modifications to typical azure-sru script to validate https://github.com/canonical/cloud-init/pull/539